ports/atmel-samd/boards/adafruit_sproutsense_m0/pins.c

```diff
@@ -0,0 +1,17 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_LIGHT), MP_ROM_PTR(&pin_PA00) },
+    { MP_ROM_QSTR(MP_QSTR_NEOPIXEL), MP_ROM_PTR(&pin_PA01) },
+
+    { MP_ROM_QSTR(MP_QSTR_EXCITATION), MP_ROM_PTR(&pin_PA01) },
+    { MP_ROM_QSTR(MP_QSTR_SENSE), MP_ROM_PTR(&pin_PA01) },
```

Comment:
Should these all be `PA01`?
![image](https://github.com/adafruit/circuitpython/assets/8755041/0d0e2a99-f812-429f-b483-87a8543c5aca)

Reply:
**NOTE** - I think the above is not current. So can't use for actual pins. But still same comment about multiple `PA01` entries.

Reply:
yup that is just wrong